Bottom: 210eba9cc201da9abbf865e0f226eea5c6f5b402
Top:    c4ccc5e8023c3fdd52548e3e38657ac71cf3c693
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-02-24 02:08:31 -0300

Refresh of debug-aliases-and-trampolines-inc.patch

---

diff --git a/gcc/vmsdbgout.c b/gcc/vmsdbgout.c
index fc87828..40f6d41 100644
--- a/gcc/vmsdbgout.c
+++ b/gcc/vmsdbgout.c
@@ -168,6 +168,8 @@ static void vmsdbgout_early_global_decl (tree);
 static void vmsdbgout_late_global_decl (tree);
 static void vmsdbgout_type_decl (tree, int);
 static void vmsdbgout_abstract_function (tree);
+static void vmsdbgout_aliased_decl (tree, tree);
+static void vmsdbgout_trampoline_decl (tree, tree);
 
 /* The debug hooks structure.  */
 
@@ -198,8 +200,8 @@ const struct gcc_debug_hooks vmsdbg_debug_hooks
    debug_nothing_tree_tree_tree_bool, /* imported_module_or_decl */
    debug_nothing_tree,		  /* deferred_inline_function */
    vmsdbgout_abstract_function,
-   debug_nothing_tree_tree,	  /* aliased_decl */
-   debug_nothing_tree_tree,	  /* trampoline_decl */
+   vmsdbgout_aliased_decl,	  /* aliased_decl */
+   vmsdbgout_trampoline_decl,	  /* trampoline_decl */
    debug_nothing_rtx_code_label,  /* label */
    debug_nothing_int,		  /* handle_pch */
    debug_nothing_rtx_insn,	  /* var_location */
@@ -1551,6 +1553,24 @@ vmsdbgout_abstract_function (tree decl)
     (*dwarf2_debug_hooks.outlining_inline_function) (decl);
 }
 
+/* Not implemented in VMS Debug.  */
+
+static void
+vmsdbgout_aliased_decl (tree alias, tree decl)
+{
+  if (write_symbols == VMS_AND_DWARF2_DEBUG)
+    (*dwarf2_debug_hooks.aliased_decl) (alias, decl);
+}
+
+/* Not implemented in VMS Debug.  */
+
+static void
+vmsdbgout_trampoline_decl (tree trampoline, tree decl)
+{
+  if (write_symbols == VMS_AND_DWARF2_DEBUG)
+    (*dwarf2_debug_hooks.trampoline_decl) (trampoline, decl);
+}
+
 /* Output stuff that Debug requires at the end of every file and generate the
    VMS Debug debugging info.  */
